managed/services/inventory/grpc/agents_server.go

```diff
@@ -227,6 +236,8 @@ func (s *agentsServer) ChangeAgent(ctx context.Context, req *inventoryv1.ChangeA
 		return s.s.ChangeMongoDBExporter(ctx, agentID, req.GetMongodbExporter())
 	case *inventoryv1.ChangeAgentRequest_PostgresExporter:
 		return s.s.ChangePostgresExporter(ctx, agentID, req.GetPostgresExporter())
+	case *inventoryv1.ChangeAgentRequest_ValkeyExporter:
+		return nil, errors.New("Valkey Exporter is not supported yet")
```

Comment:
fmt.Errorf?

Reply:
Changed.

Reply:
seems like I was wrong with this suggestion, linter in another PR I was working suggested to use errors.New if there are no variables.